apps/perf-test-react-components/src/scenarios/FluentProviderWithTheme.tsx

```diff
@@ -5,9 +5,11 @@ import { teamsLightTheme } from '@fluentui/react-theme';
 const LayoutShift: React.FunctionComponent = ({ children }) => {
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28887&commitId=62fff81864083447b7d0fa4ac4f320be73cc9711&env=prod&clientType=FLUENTUI)


  <details><summary><b>Datepicker</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Datepicker.Datepicker Example Today Teams.Shows calendar with hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.Shows datepicker popup through button..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.Shows selected date in calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.Shows selected date in input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams High Contrast.Shows datepicker popup.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams High Contrast.Shows selected date.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams High Contrast.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>